.github/workflows/comment-on-e2e-check.yaml

```
@@ -0,0 +1,108 @@
+    steps:
+      - name: Download artifact
+        id: artifact-download
+        uses: actions/github-script@v7
+        with:
+          script: |
+            let allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
+               owner: context.repo.owner,
+               repo: context.repo.repo,
+               run_id: context.payload.workflow_run.id,
+            });
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)